manifests/n/NordSecurity/NordLayer/3.2.3.0/NordSecurity.NordLayer.installer.yaml

```
@@ -0,0 +1,19 @@
+# Created with Komac v1.11.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.5.0.schema.json
+
+PackageIdentifier: NordSecurity.NordLayer
+PackageVersion: 3.2.3.0
+InstallerLocale: en-US
+InstallerType: exe
+Scope: machine
+InstallerSwitches:
+  Silent: /qn /norestart
+  SilentWithProgress: /qn /norestart
+  Custom: ''

```

```suggestion
InstallerType: burn
Scope: machine
```
When a package is just an MSI wrapped in an EXE like this, most of the time it is a `burn` installer. In the few times that it isn't, using `/qb /norestart` would be more appropriate for the `SilentWithProgress` switch, as it will show the progress bar while still performing the unattended install. If it also writes an `msiexec` command in the `UninstallString` in registry, then `AppsAndFeaturesEntries` will also be needed to specify the installer type as msi, or else users will get the message that install technologies differ anytime they attempt an upgrade

I tested this locally by checking out the branch, and `burn` seems to work with no additional switches.